integrations/slack/src/const.ts

```diff
@@ -1 +1,4 @@
 export const INTEGRATION_NAME = 'slack'
+
+export const senderIdTag = `${INTEGRATION_NAME}:senderId` as const
```

Comment:
Do you mean the variable name or the actual value stored in db? If the value, it's already prefixed by the integration name.
If the variable, we are in the slack directory making it clear enough already no?

Reply:
Oh yes, sorry, forgot it's already prefixed with "slack". What I meant is just using the original term of the provider, so for example if Slack calls it a "user" rather than a "sender", we call it "userId".